File: examples/openapi-sdk-petstore/go/sdk/query_style.go
package petstore

// QueryStyle controls how array-valued query parameters are
// serialized when explode is false. Matches the OpenAPI 3 `style`
// field. The default is QueryStyleForm with explode=true.
type QueryStyle int

const (
	QueryStyleForm QueryStyle = iota
	QueryStyleSpaceDelimited
	QueryStylePipeDelimited
)

File: examples/openapi-sdk-petstore/kotlin/sdk/com/example/petstore/api/QueryStyle.kt
package com.example.petstore.api

/**
 * Query-array serialization style — matches the OpenAPI 3 `style` field
 * for query parameters whose value is an array. The default is `form`
 * with explode=true; the others appear when the spec opts into them.
 */
public enum class QueryStyle {
    FORM,
    SPACE_DELIMITED,
    PIPE_DELIMITED,
}
